docs/workflows.md

```diff
@@ -11,8 +11,8 @@ These are onetime configuration steps required to set up the GitHub Actions work
 *Required repository secrets for the CI/CD.*
 | Secret name | Description |
 | ----------- | ----------- |
-| `AZURE_CREDENTIALS` | Explained in [Bootstrapping - Create service principals](./bootstrapping.md#create-service-principals). |
-| `AZURE_CONTRIBUTOR_SP` | Explained in [Bootstrapping - Create service principals](./bootstrapping.md#create-service-principals). |
+| `AZURE_CREDENTIALS` | Explained in [Bootstrapping - Create service principals](./bootstrapping.md#create-service-principals). Main service principal credentials output. |
+| `AZURE_CONTRIBUTOR_SP` | Explained in [Bootstrapping - Create service principals](./bootstrapping.md#create-service-principals). Resource Processor Function service principal credentials output.|
```

Comment:
Is it still required given we don't use a resource processor function anymore (moved to vmss and managed identities)?

Reply:
We are going to move the resource processor function out of the repo into a separate branch - in case we need it later. With this in mind, let's move the processor function specifics out of the docs.